dask/bytes/tests/test_compression.py

```diff
@@ -0,0 +1,39 @@
+from io import BytesIO
+
+import pytest
+from dask.bytes.compression import compress, decompress, files
+
+def test_compression():
+    assert set(compress) == set(decompress)
+
+    a = b'Hello, world!'
+    for k in compress:
+        comp = compress[k]
+        decomp = decompress[k]
+        b = comp(a)
+        c = decomp(b)
+        assert a == c
+        if k is not None:
+            assert a != b
```

Comment:
For some compression, I could imagine cases where the compressed output is identical to the input (e.g., the text is not the sort of data it was made for, and the algorithm size would be bigger than the original).